shared/package.json

```diff
@@ -3,45 +3,39 @@
   "version": "1.0.1",
   "description": "",
   "scripts": {
-    "_helper": "yarn run _node -- desktop/yarn-helper.js",
+    "--[helpers]--": "--------------------------------------------------------",
```

Comment:
i just made some comment nodes so we can divide all the things the package.json does